mne/bem.py

```diff
@@ -332,3 +337,108 @@ def cost_fun(x, points):
     origin = x_opt[:3]
     radius = x_opt[3]
     return radius, origin
+
+
+# ############################################################################
+# Create BEM surfaces
+
+@verbose
+def make_watershed_bem(subject=None, subjects_dir=None, overwrite=False,
```

Comment:
Yeah, looking for `SUBJECTS_DIR` in the env is fine because we do that everywhere in the package. Same thing with FREESURFER_HOME, we look for that in the environment elsewhere as well. I'm glad `subject` is required, I think it's safer. Sounds good to me.